htdocs/api/coord.ch/newest.php

```diff
@@ -7,18 +7,18 @@
 
 $opt['rootpath'] = '../../';
 header('Content-type: text/html; charset=utf-8');
-require($opt['rootpath'] . 'lib2/web.inc.php');
+require(__DIR__ . '/../../lib2/web.inc.php');
 
 $bFirstRow = true;
 
 $rs = sql(
-    "SELECT SQL_BUFFER_RESULT `caches`.`wp_oc` , `caches`.`name` , `user`.`user_id` , `user`.`username`
+    'SELECT SQL_BUFFER_RESULT `caches`.`wp_oc` , `caches`.`name` , `user`.`user_id` , `user`.`username`
      FROM `caches` 
      INNER JOIN `user` ON `caches`.`user_id` = `user`.`user_id`
      INNER JOIN `cache_status` ON `caches`.`status` = `cache_status`.`id`
      WHERE `cache_status`.`allow_user_view` = 1 
      ORDER BY `caches`.`date_created` 
-     DESC LIMIT 3"
+     DESC LIMIT 3'
```

Comment:
you can refactor it again if you change the sql statement. We have some more examples with single quote and we should not invest more time because plain sql statements will be replaces with DBAL or Doctrine QueryBuilder in the future.